js/components/home.tsx

```diff
@@ -54,11 +59,14 @@ export const Home = (): ReactElement => {
       </section>
       <section className="max-w-lg mx-auto px-2 py-5">
         <p className="font-semibold mb-3">Today&apos;s sign ins</p>
-        <List line="blue" />
+        <List key={signInCount} line="blue" />
```

Comment:
I spent a couple of hours trying to test _specifically_ that the state increment causes the list to reload. It's theoretically possible, but I got into a mess of mocking the modal and trying to trigger `onComplete` with a fake button. I'm submitting this PR for now and I can re-visit if we think it's absolutely necessary.

Reply:
I'd be curious to learn / chat more about the approach you were trying and what issues you were running into.

Reply:
(We talked on Slack)
This is a lot better! bb2b335